cytominer_eval/utils/hitk_utils.py

```diff
@@ -0,0 +1,72 @@
+def add_hits_hits(df):
```

Comment:
ah! I think this function makes it clear to me what an "index" is. It is just the rank!

This clears things up for me.

I do, however, think that this function name can be improved. Instead of `add_hits_hits`, how about `add_hit_rank`, or something more specific